src/client/view/movieList/movieList.test.jsx

```diff
@@ -0,0 +1,54 @@
+
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import MovieList from './movieList';
+import MovieItem from '../movieItem/movieItem';
+
+describe('MovieList component', () => {
+
+    let data;
+    let onClick;
+    beforeAll(() => {
+        data = [{
```

Comment:
I would suggest to create a file with movie mocks. In all your tests you may then just invoke a method like:
`getMockedMovies(numberOfMovies)`

Reply:
this is not crucial at all, btw